test/test_models.py

```diff
@@ -830,6 +833,8 @@ def test_video_model(model_fn, dev):
         "num_classes": 50,
     }
     model_name = model_fn.__name__
+    if dev == "cuda" and SKIP_BIG_MODEL and model_name in skipped_big_models:
```

Comment:
We should change this to:
```suggestion
    if and SKIP_BIG_MODEL and model_name in skipped_big_models:
```

If #6197 gets merged.